<launch>
  <master auto="start"/>


  <!-- Run the map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find nao_2dnav)/map.yaml" respawn = "false">
    <param name ="frame_id" value="map" />
  </node>


  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="1 0 0 0 0 0 1 map nav_odometry 100" />


  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find nao_2dnav)/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find nao_2dnav)/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find nao_2dnav)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find nao_2dnav)/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find nao_2dnav)/base_local_planner_params.yaml" command="load" />
  </node>

  <!-- navigation stack related -->
  <!--fake localization for navigation -->
  <!--node pkg="fake_localization" type="fake_localization" name="fake_localization" clear_params="true" output="screen">
      <param name="global_frame_id" value="nav_odometry" />
      <param name="base_frame_id" value="nav_base_link" />
  </node-->


</launch>
